_changelog/entries/2021/2021-03-12-pendo-v1-ijson-performance-enhancement.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Pendo (v1) improvement: using ijson to improve `visitors` and `events` replication
```

Comment:
```suggestion
title: "Pendo (v1) improvement: Improve performance for`visitors` and `events` tables"
```

My thought here (aside from the double quotes) was to make this general and relatable. Mentioning how this improvement works is good in the body, but here it might not have as much impact if someone is skimming.